README.md

```diff
@@ -0,0 +1,43 @@
+**Instructions**<br>
```

Comment:
Change to "About".
Should use the same title style as the below ones.
Add a section "Used technologies", and list all tools you have used: Java version, Docker, AWS, etc.